requirements.txt

```diff
@@ -16,7 +16,8 @@ netifaces==0.10.7
 networkx==2.1.0
 psutil==5.4.5
 pycryptodome==3.6.1
-raiden-libs==0.1.2
+# raiden-libs==0.1.2
+git+https://github.com/andrevmatos/raiden-libs@c689c2396173e1fbbc9d53758fd3c0751bf82dd4
```

Comment:
Raiden-libs master is not yet compatible with raiden client due to [this](https://github.com/raiden-network/raiden-libs/pull/116) PR. We are still working on making the client compatible with that.